### PR TITLE
add ansible-security-integration-qradar-python38

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

File: zuul.d/project.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@
       jobs:
         - ansible-cloud-vcenter-appliance:
             voting: false
+        - ansible-security-integration-qradar-python38:
```
Comment (pabelanger) [inline zuul.d/project.yaml:9]: be sure to also add this to gate

File: zuul.d/project.yaml [hunk 2]
```diff
@@ -6,6 +6,10 @@
...
+            files:
+              - playbooks/ansible-test-network-integration-base/files/bootstrap-qradar.yaml
```
Comment (pabelanger) [inline zuul.d/project.yaml:11]: playbooks/ansible-network-appliance-base/.*

we want to trigger a job run here, is we change the base playbooks

Comment (goneri) [reply zuul.d/project.yaml]: Oky, thanks for the feedback @pabelanger.

File: zuul.d/project.yaml [hunk 3]
```diff
@@ -44,6 +44,11 @@
             files:
               - playbooks/ansible-network-appliance-base/.*
               - playbooks/ansible-network-vyos-appliance/.*
+        - ansible-security-integration-qradar-python38:
+            files:
+              - playbooks/ansible-test-network-integration-base/files/bootstrap-qradar.yaml
+              - playbooks/ansible-network-appliance-base/.*
+              - zuul.d/ansible-security-jobs.yaml
```
Comment (pabelanger) [inline zuul.d/project.yaml:51]: there is no need to match on this file, zuul does it by default

Comment (goneri) [reply zuul.d/project.yaml:51]: ok, interesting.

File: zuul.d/project.yaml [hunk 4]
```diff
@@ -44,6 +44,11 @@
             files:
               - playbooks/ansible-network-appliance-base/.*
               - playbooks/ansible-network-vyos-appliance/.*
+        - ansible-security-integration-qradar-python38:
+            files:
```
Comment (pabelanger) [inline zuul.d/project.yaml:48]:
```
files:
  - playbooks/ansible-network-appliance-base/.*
```
should be enough

**Comment by goneri** [comment]
recheck

**Pull request merged**